plans/001-readme-and-getting-started.md

```
@@ -0,0 +1,155 @@
+
+
+  WIP
+  ```
+
+Facts to use in the content (verified against the repo at commit `db64bb8`):
+
+- Project: a shadcn-vue registry of extra components and blocks that are not in the official shadcn-vue collection. Docs site: https://extended.shadcn-vue.com. Repo: https://github.com/unovue/shadcn-vue-extended (both values live in `nuxt.config.ts` `appConfig`).
+- Install command pattern (used throughout existing docs, e.g. `content/docs/2.components/separator-label.md`):
+  `npx shadcn-vue@latest add https://extended.shadcn-vue.com/r/<name>.json`
+- Current registry items (from `registry.json`):
+  | Name | Type | What it is |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)